### ISSUE TITLE
The server principal is not able to access the database "model" under the current security context

### ISSUE THREAD

**Issue by sadedil** [Microsoft/azuredatastudio#3840]
With the latest insiders version, i can't connect the databases which i've previously able to connect.

It says *The server principal "MYDOMAIN\MyUserName" is not able to access the database "model" under the current security context.*

- Azure Data Studio Version: [February Insiders Build 1.4.3](https://github.com/Microsoft/azuredatastudio/releases/tag/1.4.3)
- Setup File: azuredatastudio-windows-setup-1.4.3.exe

Steps to Reproduce:

1. Download and install *Azure Data Studio February Insiders Build 1.4.3* from the releases page (link above)
2. Try to connect a database which is previously able to connect with Windows Authentication
3. Get this error: *The server principal "MYDOMAIN\MyUserName" is not able to access the database "model" under the current security context.*

Exceptions:

1. I am able to connect an SQL Server Express LocalDB instance

Workarounds:

1. Rollback to the latest stable version (January Hotfix 2019 (version 1.3.9)) solves the problem

Detailed output:
```
System.Data.SqlClient.SqlException (0x80131904): The server principal "MYDOMAIN\MyUserName" is not able to access the database "model" under the current security context.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, SqlDataReader ds)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   at Microsoft.SqlTools.ServiceLayer.Connection.ReliableConnection.ReliableConnectionHelper.ExecuteReader(IDbConnection conn, String commandText, Action`1 readResult, Action`1 initializeCommand, Predicate`1 catchException) in D:\a\1\s\src\Microsoft.SqlTools.ServiceLayer\Connection\ReliableConnection\ReliableConnectionHelper.cs:line 393
   at Microsoft.SqlTools.ServiceLayer.Connection.ReliableConnection.ReliableConnectionHelper.<>c__DisplayClass22_0.<IsSqlDwDatabase>b__0(String commandText) in D:\a\1\s\src\Microsoft.SqlTools.ServiceLayer\Connection\ReliableConnection\ReliableConnectionHelper.cs:line 496
   at Microsoft.SqlTools.ServiceLayer.Connection.ReliableConnection.ReliableConnectionHelper.IsSqlDwDatabase(IDbConnection connection) in D:\a\1\s\src\Microsoft.SqlTools.ServiceLayer\Connection\ReliableConnection\ReliableConnectionHelper.cs:line 508
   at Microsoft.SqlTools.ServiceLayer.Connection.ReliableConnection.ReliableSqlConnection.IsSqlDwConnection(IDbConnection conn) in D:\a\1\s\src\Microsoft.SqlTools.ServiceLayer\Connection\ReliableConnection\ReliableSqlConnection.cs:line 119
   at Microsoft.SqlTools.ServiceLayer.Connection.ReliableConnection.ReliableSqlConnection.<>c__DisplayClass28_0.<<OpenAsync>b__0>d.MoveNext() in D:\a\1\s\src\Microsoft.SqlTools.ServiceLayer\Connection\ReliableConnection\ReliableSqlConnection.cs:line 304
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.SqlTools.ServiceLayer.Connection.ConnectionService.TryOpenConnection(ConnectionInfo connectionInfo, ConnectParams connectionParams) in D:\a\1\s\src\Microsoft.SqlTools.ServiceLayer\Connection\ConnectionService.cs:line 521
ClientConnectionId:fc945b21-f9de-4212-9b77-171038106752
Error Number:916,State:1,Class:14
```

**Comment by enecciari** [comment]
I think is a sqltoolservice issue... upgrading it solve the problem

**Comment by zengshengliu** [comment]
I have the same issue today after the 1.4.3-insiders update.  How do I go about updating sqltoolservice on the Mac OS?

**Comment by enecciari** [comment]
> I have the same issue today after the 1.4.3-insiders update. How do I go about updating sqltoolservice on the Mac OS?

	I opened /Applications/Azure Data Studio.app/Contents/Resources/app/extensions/mssql/sqltoolsservice/OSX/1.5.0-alpha.67
and upgraded the files.

**Comment by zengshengliu** [comment]
> > I have the same issue today after the 1.4.3-insiders update. How do I go about updating sqltoolservice on the Mac OS?
> 
> ```
> I opened /Applications/Azure Data Studio.app/Contents/Resources/app/extensions/mssql/sqltoolsservice/OSX/1.5.0-alpha.67
> ```
> and upgraded the files.

Thank you.
Just a FYI for others, I use the following version of the sql tools service https://github.com/Microsoft/sqltoolsservice/releases/tag/v1.5.0-alpha.6

**Comment by SebastianPfliegel** [comment]
Had the same issue and updating the sqltoolsservice did the trick as well ;)

**Comment by zengshengliu** [comment]
Looks like it is fixed in 1.4.4-Insider update that I received earlier today.
(they updated the SQL Tools Service to 1.4.0-alpha.69)

**Comment by kburtram** [comment]
(@YurongHe ) This was probably related to this change to support Big Data connections https://github.com/Microsoft/sqltoolsservice/commit/adc13cff8209981f7066ecc8aa3c1e4554d5ea8d.  

Sorry for the inconvenience.  I'll close the issue since it's resolved in 1.4.4.  Thanks for reporting.
